k8s/aat/common/reform-scan/notification-service.yaml

```diff
@@ -43,6 +45,7 @@ spec:
           SLACK_CHANNEL: "bsp-test-notices"
           SLACK_NOTIFY_SUCCESS: false
           TEST_S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+          PENDING_NOTIFICATIONS_SEND_DELAY_IN_MINUTE: 0
```

Comment:
this is needed in the app not for func tests ?

Reply:
yes.